lib/indexers/projects-content/index.js

```diff
@@ -0,0 +1,184 @@
+const { pick, get } = require('lodash');
+const extractContent = require('./extract-content');
+
+const indexName = 'projects-content';
+const columnsToIndex = [
+  'id',
+  'title',
+  'status',
+  'licenceNumber',
+  'schemaVersion'
+];
+
+const indexProject = (esClient, project, ProjectVersion) => {
+  return ProjectVersion.query()
+    .where({
+      // look for most recent submitted draft for inactive projects
+      status: project.status === 'inactive' ? 'submitted' : 'granted',
+      projectId: project.id
+    })
+    .orderBy('updatedAt', 'desc')
+    .first()
```

Comment:
```suggestion
    .limit(1)
    .first()
```

Reply:
As far as I can tell from the docs, Objection's `first()` does not apply a limit, so all the records are returned and then it's just an array op.